shared/rakefiles/xk_util.rake

```diff
@@ -258,13 +271,16 @@ task :restore_snapshot_from_image_file, [:snapshot_files] => [@gcp_creds_file, :
   end
```

Comment:
(Continued from https://github.com/gpii-ops/gpii-infra/pull/349#discussion_r298699789)

I don't understand how running `gsutil ls` (and not capturing the output) could do useful work. What am I missing?

Reply:
the `gsutil ls` command returns non-zero code if the file is not found. I will clarify this in a comment.